infra/cifuzz/build_fuzzers_test.py

```diff
@@ -53,7 +53,7 @@
 # A fuzzer to be built in build_fuzzers integration tests.
 EXAMPLE_BUILD_FUZZER = 'do_stuff_fuzzer'
 
-# pylint: disable=no-self-use,protected-access,too-few-public-methods
+# pylint: disable=no-self-use,protected-access,too-few-public-methods,unused-argument
```

Comment:
I prefer naming params "_, __, etc." rather than enabling this. It makes the function easier to understand I think if you know which params are ignored.

Reply:
Not a big fan of _ (__, ___, ____ as we get more). The preferred approach is actually to leave these named and use "del": https://google.github.io/styleguide/pyguide.html#214-decision, but that doesn't seem necessary for a test file. 

Can change this is you still feel strongly.

Reply:
I think this is fine as is. But I prefer less pylint directives so would prefer `del blah` to this but this is OK.